extensions/wikia/PortableInfobox/tests/nodes/NodeGroupTest.php

```diff
@@ -92,9 +92,9 @@ public function groupNodeTestProvider() {
 			  [ '1' => 'one', '2' => 'two' ],
 			  [ 'value' => [
 				  [ 'type' => 'header', 'data' => [ 'value' => 'h' ], 'isEmpty' => false, 'source' => [ ] ],
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "header" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)